Source/Renderer/Context.js

```diff
@@ -508,6 +509,18 @@ define([
         },
 
         /**
+         * <code>true</code> if the WebGL context supports stencil buffers.
+         * Stencil buffers are not supported by all systems.
+         * @memberof Context.prototype
+         * @type {Boolean}
+         */
+        stencilBuffer : {
+            get : function() {
+                return this._stencilBits >= 8;
```

Comment:
I've never seen less than 8 (or anything other than 8), but are you sure this is the right test?